src/server/verification/processor/provider/ZoomProvider.js

```diff
@@ -111,17 +120,24 @@ class ZoomProvider implements IEnrollmentProvider {
         // so we'll reject with isNotMatch: true instead
         // to show the error screen on app side immediately
         // notifying about liveness check failed
-        throwCustomException(message, { isNotMatch }, response)
+        throwException(message, { isNotMatch }, response)
```

Comment:
would the response include the blob? we dont need it to be logged in errors

Reply:
@sirpy throwException filters any blob / base64 fields from logging